servlet/5.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Servlet 5.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Servlet 5.0"
+date: 2020-07-20
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
 
 * [Jakarta Servlet 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/5.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0-SNAPSHOT.html) (HTML)
+* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0.pdf) (PDF)
+* [Jakarta Servlet 5.0 Specification Document](./servlet-spec-5.0.html) (HTML)
 * [Jakarta Servlet 5.0 Javadoc](./apidocs)
-* [Jakarta Servlet 5.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Servlet 5.0 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-servlet-tck-5.0.0.zip)([sig](),[sha](),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
This TCK url should be the "final" location for the TCK.  Along the lines of https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.0.zip.  Also, don't forget to include the sig and sha links.

Reply:
Yes, I know, although there's nothing there yet. Only https://download.eclipse.org/jakartaee/servlet/4.0/ 

Who is responsible for putting the TCK under /5.0?

Reply:
That's part of the final approval process after the ballot is complete.  A member of the Spec Committee promotes the TCK to the proper location.  The requirement at this point is to put the expected location in the _index.md file and to provide the staging version of the TCK in the staging repo (which you already did).

Reply:
This looks better now. Thanks.  But, we still need the sig and sha links as well...

Reply:
Thanks, done!